src/ds5/ds5-options.h

```diff
@@ -478,4 +478,23 @@ namespace librealsense
 
         std::function<void(const option&)> _recording_function = [](const option&) {};
     };
+
+    /*class safety_preset_option : public option
```

Comment:
Not sure we need a special option.
Does the SRS indicates some conditions about when we are not allowed to switch preset?